### PR TITLE
Added good permutation

### LINKED ISSUE #1297: Good Permutations in C, C++, Java, Python & Go

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #1297

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added implementation for Good permutation in 5 languages

#### Languages Used:
- C
- C++
- Python
- Java
- Go

#### Files Added:
- Good_Permutation.c
- Good_Permutation.cpp
- Good_Permutation.py
- Good_Permutation.java
- Good_Permutation.go

### CODE REVIEW

File: Good_Permutation/Good_Permutation.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+        GOOD PERMUTATION
```
Comment (marwahmanbir) [inline Good_Permutation/Good_Permutation.go]: Remove indentation from comment.

File: Good_Permutation/Good_Permutation.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
+        GOOD PERMUTATION
```
Comment (marwahmanbir) [inline Good_Permutation/Good_Permutation.java]: Remove indentation from comment.

File: Good_Permutation/Good_Permutation.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+'''
+        GOOD PERMUTATION
```
Comment (marwahmanbir) [inline Good_Permutation/Good_Permutation.py]: Remove indentation from comment.

**Comment by MastersAbh** [comment]
Done. Please review @marwahmanbir

**Comment by MastersAbh** [comment]
@marwahmanbir  please review.

**Comment by MastersAbh** [comment]
@marwahmanbir  please have a look at this PR.

**Pull request merged**